src/Events.Functions/Clients/OutboundClient.cs

```diff
@@ -6,54 +6,54 @@
 using System.Text.Json;
 using System.Threading.Tasks;
 using Altinn.Common.AccessTokenClient.Services;
+using Altinn.Platform.Events.Functions.Clients.Interfaces;
 using Altinn.Platform.Events.Functions.Configuration;
 using Altinn.Platform.Events.Functions.Extensions;
 using Altinn.Platform.Events.Functions.Models;
 using Altinn.Platform.Events.Functions.Services.Interfaces;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
-namespace Altinn.Platform.Events.Functions.Services
+namespace Altinn.Platform.Events.Functions.Clients
 {
     /// <summary>
     /// Handles PushEvents service
     /// </summary>
-    public class PushEventsService : IPushEventsService
+    public class OutboundClient : IOutboundClient
     {
         private readonly HttpClient _client;
         private readonly IAccessTokenGenerator _accessTokenGenerator;
         private readonly IKeyVaultService _keyVaultService;
         private readonly KeyVaultSettings _keyVaultSettings;
-        private readonly PlatformSettings _platformSettings;
-        private readonly ILogger<IPushEventsService> _logger;
+        private readonly ILogger<IOutboundClient> _logger;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="PushEventsService"/> class.
+        /// Initializes a new instance of the <see cref="OutboundClient"/> class.
         /// </summary>
-        public PushEventsService(
+        public OutboundClient(
             HttpClient httpClient,
             IAccessTokenGenerator accessTokenGenerator,
             IKeyVaultService keyVaultService,
             IOptions<PlatformSettings> eventsConfig,
             IOptions<KeyVaultSettings> keyVaultSettings,
-            ILogger<IPushEventsService> logger)
+            ILogger<IOutboundClient> logger)
         {
-            _platformSettings = eventsConfig.Value;
+            var platformSettings = eventsConfig.Value;
+            httpClient.BaseAddress = new Uri(platformSettings.ApiEventsEndpoint);
             _keyVaultSettings = keyVaultSettings.Value;
-            httpClient.BaseAddress = new Uri(_platformSettings.ApiEventsEndpoint);
             _client = httpClient;
             _accessTokenGenerator = accessTokenGenerator;
             _keyVaultService = keyVaultService;
             _logger = logger;
         }
 
         /// <inheritdoc/>
-        public async Task SendToPushController(CloudEvent item)
+        public async Task PostOutbound(CloudEvent item)
         {
             StringContent httpContent = new(JsonSerializer.Serialize(item), Encoding.UTF8, "application/json");
             try
             {
-                string endpointUrl = "push";
+                string endpointUrl = "outbound";
```

Comment:
You've changed the output here.. Meaning we risk that the endpoint isn't available when the functions is deployed. It is my strong recommendation that all changes related to Events.Functions is removed from this PR and handled separately.